metrics/record.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+	"path"
+
+	"github.com/knative/pkg/logging"
+	"github.com/knative/pkg/metrics/metricskey"
+	"go.opencensus.io/stats"
+)
+
+// Record decides whether to record one measurement based on current
+// metrics config. If yes, it records measurements via OpenCensus.
+func Record(ctx context.Context, ms stats.Measurement) {
+	mc := getCurMetricsConfig()
+	if mc == nil {
+		logger := logging.FromContext(ctx)
```

Comment:
join these two lines?
Also is it useful at all?

Reply:
Which two lines?

I just want to send this log for debugging purpose in case this function is called before the exporter is initialized..

Reply:
`logging.FromContext(ctx).Error(....)`.
And what I mean is: is it useful to log this at all?

Reply:
I think it is useful in case we don't see the expected metrics. If we pass sufficient information in `ctx`, the logs should include extra information. I changed this log from `Error` to `Warn`.